pkg/generator/ingress_translator.go

```diff
@@ -48,11 +48,13 @@ import (
 
 type translatedIngress struct {
 	name                    types.NamespacedName
-	sniMatches              []*envoy.SNIMatch
+	localSNIMatches         []*envoy.SNIMatch
```

Comment:
Thinking out loud is there a scenario where a ksvc wants to communicate with an egress target via the knative local gateway and the target does not need mutual tls. I was wondering about the use of [auto_sni](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/protocol.proto#envoy-v3-api-field-config-core-v3-upstreamhttpprotocoloptions-auto-sni).

Reply:
Hm sounds to me more like a Service-Mesh case. I don't think we ever want to go through kourier for egress traffic, right?